validate/userMiddleware.js

```diff
@@ -0,0 +1,17 @@
+const passport = require("passport");
+
+const authMiddleware = async (req, res, next) => {
+return passport.authenticate("jwt", { session: false }, (error, user) => {
+    if (!user || error) {
```

Comment:
Sprawdź jeszcze czy uzytkownik ma aktywny token user.token
```suggestion
    if (!user || error || !user.token) {
```